src/index.html

```diff
@@ -6,9 +6,26 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#"><img src="images/logo.png" alt="MoYo"></a>
+      </div>
+      <nav class="meny">
+        <ul class="gadges">
+          <li data-qa="hover" class="is-active"><a href="#####">Apple</a></li>
+          <li data-qa="hover" class="is-active"><a href="#####">Samsung</a></li>
```

Comment:
Class is-active should be only on first a element
![image](https://github.com/mate-academy/layout_moyo-header/assets/108701488/3b3706c2-729a-4185-9b6f-1d676bb61e70)

Reply:
The same is for other elements.